specs/protocol/abi.md

```diff
@@ -66,7 +66,20 @@ Here an example containing custom types:
                                 "type": "[u8; 2]"
                             }
                         ]
-                    }
+                    },
+                    {
+                        "name": "z",
+                        "type": "tuple (u16, u8)",
```

Comment:
Is this how `forc` will spit out enum types?

Reply:
Yes, maybe `tuple` is enough? It may not be descriptive enough compared to `tuple (u16, u8)`.  I don't have a strong opinion either way.